src/resource_utils/sqs_utils.js

```diff
@@ -25,9 +25,7 @@ function getSNSTrigger(messages) {
                 'MessageId' in body &&
                 'TopicArn' in body &&
                 'Message' in body &&
-                'Timestamp' in body &&
-                'SignatureVersion' in body &&
```

Comment:
Why did you remove them?
someone else uses them?

Reply:
This is used to identify the message within SQS got the SNS signature. in SNS FIFO both signature fields do not appear and that's why they are not crucial (TopicArn, MessageId, and the rest are a good indicator for SNS usage as well)